### PR TITLE
Better long running event logging

### LINKED ISSUE #2030: Allow for finding long-running events outside of `TRACE` debug levels

### PR DESCRIPTION
**Pull request by marc-casperlabs**
Closes #2030 

Adds a fixed string to the log of long running events.

### CODE REVIEW

File: node/src/reactor.rs [hunk 1]
```diff
@@ -283,6 +283,11 @@ pub(crate) trait ReactorEvent: Send + Debug + From<ControlAnnouncement> + 'stati
     /// [`ControlAnnouncement`](`crate::effect::announcements::ControlAnnouncement`) if the event
     /// is indeed a control announcement variant.
     fn as_control(&self) -> Option<&ControlAnnouncement>;
+
+    /// Returns a cheap but human-readable description of the event.
+    fn description(&self) -> &'static str {
+        "anonymous event"
```
Comment (goral09) [inline node/src/reactor.rs:289]:
```suggestion
        "unknown event type"
```
?

**Review by goral09 — APPROVED** [review]
Welcomed improvement.

File: node/src/reactor.rs (hunk 1 above)
Comment (marc-casperlabs) [reply node/src/reactor.rs:289]: It's not exactly unknown though? :)

Comment (marc-casperlabs) [reply node/src/reactor.rs:289]: I wanted something that does not make the user give up immediately, but rather realize that there might be something that one can do about it!

**Review by afck — APPROVED** [review]

**Pull request merged**